Include.psm1

```diff
@@ -563,7 +562,6 @@ function Get-Stat {
                 $Stat = Get-Content "Stats\$Stat_Name.txt" -ErrorAction Stop | ConvertFrom-Json -ErrorAction Stop
                 $Global:Stats | Add-Member @{ 
                     $Stat_Name = [PSCustomObject]@{ 
-                        Name                  = [String]$Stat_Name
```

Comment:
Mistake???

Reply:
This change is, again, rejected and will be removed at 12 noon GMT: 
This breaks the foundational idea as to how objects should be defined in programming languages. The way in which objects and arrays are both returned in PowerShell functions is also a distinguishing feature. To return this is without the name is fundamentally wrong and makes returning multiple stats from the function unusable.